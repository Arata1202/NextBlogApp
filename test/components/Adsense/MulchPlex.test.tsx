import { render } from '@testing-library/react';
import Mulchplex from '@/components/Adsense/MulchPlex';

describe('MulchPlex', () => {
  test('スナップショット（MulchPlex)', () => {
    const { asFragment } = render(<Mulchplex slot="" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
