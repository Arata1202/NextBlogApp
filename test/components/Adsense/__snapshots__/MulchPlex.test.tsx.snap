// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MulchPlex スナップショット（MulchPlex) 1`] = `
<DocumentFragment>
  <div
    class="FirstAd"
    style="max-width: 100%; overflow: hidden;"
  >
    <p
      class="text-center"
    >
      スポンサーリンク
    </p>
    <ins
      class="adsbygoogle"
      data-ad-client="ca-pub-1705865999592590"
      data-ad-format="autorelaxed"
      data-ad-slot=""
      data-full-width-responsive="true"
      style="display: block; width: 100%;"
    />
  </div>
</DocumentFragment>
`;
